flume/doc/quick_start_guide.md

```diff
@@ -1 +1,154 @@
-TBD
+#Cygnus Quick Start Guide
+This quick start overviews the steps a newbie programmer will have to follow in order to get familiar with Cygnus and its basic functionality. For more detailed information, please refer to the [Cygnus GitHub](https://github.com/telefonicaid/fiware-connectors).
+
+##Basic knowlegde about Cygnus
+Previous to installing and using it, you should know a couple of things about Cygnus.
+
+###What is is for?
+Cygnus is a connector in charge of persisting Orion context data in certain configured third-party storages, creating an historical view of such data. In other words, Orion only stores the last value regarding an entity's attribute, and if an older value is required then you will have to persist it in other storage, value by value, using Cygnus.
+
+###How does it receives context data from Orion Context Broker?
+Cygnus uses the subscription/notification feature of Orion. A subscription is made in Orion on behalf of Cygnus, detailing which entities we want to be notified for when an update occurs on any of those entities attributes.
+
+###Which storages is it able to integrate?
+Current stable release (0.7.1) is able to persist Orion context data in:
```

Comment:
Minor typo (sorry, I am not able to select the right line (8)): an historic -> a historic